tests/worker_deployment_version_test.go

```diff
@@ -109,6 +109,7 @@ func (s *DeploymentVersionSuite) SetupTest() {
 }
 
 func (s *DeploymentVersionSuite) TearDownTest() {
+	s.FunctionalTestBase.TearDownTest()
```

Comment:
here and all other similar lines: should we do this after stopping the worker+client?

Reply:
Good point.